papaparse.js

```diff
@@ -582,6 +582,19 @@
 
 			xhr.open('GET', this._input, !IS_WORKER);
 
+			if (this._config.downloadRequestHeaders) 
```

Comment:
I find it more logical to set it after withCredentials as we are setting up the request there.